Makefile

```diff
@@ -14,6 +14,7 @@ install-dependencies:
 	go install github.com/goreleaser/goreleaser@$(GORELEASER_VERSION)
 	go install golang.org/x/tools/cmd/godoc@latest
 	go install golang.org/x/tools/cmd/stringer@latest
+	go install github.com/google/go-licenses@latest
```

Comment:
Move to GitHub ones.

Actually we could move `golang.org/x/tools` to to top and separate the groups.